scripts/util/Centos7_only_core.sh

```diff
@@ -0,0 +1,818 @@
+#!/usr/bin/env bash
+
+#
+# Hootenanny Core Only configure and build script
+#
+# This script is basically the VagrantProvisionCentos7.sh script with the UI stuff removed. The intent is to
+# have a script that can configure a machine and then build Hootenanny either using Vagrant or straight on
+# an existing machine/VM
+#
```

Comment:
If this lives beyond initial migration to CentOS 7 we should consider combining it with `VagrantProvisionCentos7.sh` so that they stay in sync.  They aren't in sync already.